test/providers/provider_appveyorci.test.ts

```diff
@@ -59,14 +59,46 @@ describe('AppveyorCI Params', () => {
         APPVEYOR_BUILD_VERSION: '3',
         APPVEYOR_JOB_ID: '1',
         APPVEYOR_PROJECT_SLUG: 'testRepo',
-        APPVEYOR_PULL_REQUEST_NUMBER: '4',
```

Comment:
Is there a specific reason why this line was removed?

Reply:
Yes. For PRs appveyor sets e.g. `$APPVEYOR_PULL_REQUEST_HEAD_COMMIT`
So a PR run has to be tested differently with different outcome. With the (mock) variables in this test this isn't a PR build.
I basically copied this test and added the missing env var: https://github.com/codecov/uploader/pull/534/files#diff-14c6e685ddd586e988a9b18bfb02307dd44a512dc8615e90eb2bb56b2b7b35d5R83

Reply:
I don't use Appveyer myself. 🤔 

@blueyed, as a fellow Appveyor user, do you mind chiming in here? You helped a lot with the bash uploader's Appveyor support if I recall.